source/system-administrators/upgrade/upgrading-from-craftercms-3-0-17.rst

```diff
@@ -0,0 +1,17 @@
+:orphan:
+
+.. document does not appear in any toctree, this file is referenced
+   use :orphan: File-wide metadata option to get rid of WARNING: document isn't included in any toctree for now
+
+.. _upgrade-from-3-0-17:
+
+=============================================================
+Instructions for Upgrading Crafter CMS from 3.0.17 on Windows
+=============================================================
+
+For Crafter CMS installed from a bundle, before starting the upgrade process from 3.0.17, we need to download the updated scripts for the Crafter CMS environment we want to upgrade here: _put_zip_file_location_here.
+
+Next, unzip the file we just downloaded ``ENV-3.0.17-batch-file-fixes.zip`` and copy the files under ``$CRAFTER_INSTALL\bin``.
```

Comment:
We also have to say: the crafter-setenv.bat in these file fixes contains the default port. If you made changes to the ports you need also to make the same changes in the updated .bat.

Reply:
Done. Thanks!
Commit : https://github.com/craftercms/docs/pull/632/commits/0a4493931adf0ed5b255f626967b6fabeffe044d